src/integrationtest/python/features/jobs.feature

```diff
@@ -0,0 +1,12 @@
+# Created by Fenting Lin on 10/23/19
+
+Feature: # Verify functionality on jobs tab
+   # This inlcueds everything from clicking on the jobs tab
+
+  Scenario Outline: Verify the application stands up
```

Comment:
pycharm is telling me to add an example sections here, but I'm not sure if that's necessary